pvlib/solarposition.py

```diff
@@ -404,8 +406,12 @@ def sun_rise_set_transit_spa(times, latitude, longitude, how='numpy',
 
     Parameters
     ----------
-    times : pandas.DatetimeIndex
+    time : pandas.DatetimeIndex
         Must be localized to the timezone for ``latitude`` and ``longitude``.
+
+        .. versionchanged:: 0.11.2
+            Renamed from ``times`` to ``time``. Removal in ``v0.12.0``.
```

Comment:
```suggestion
            The ``times` parameter has been renamed ``time``. The deprecated
            ``times`` parameter will be removed in ``v0.12.0``.
```
I think this is a clearer